src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,143 @@
+.card-conteiner {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.img-box__img {
+  margin-top: 32px;
+  margin-left: 19px;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+}
+
+.img-box {
+  height: 166px;
+}
+
+.name {
+  color: $dark-grey;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card-conteiner__name {
+  margin-top: 40px;
+  margin-left: 16px;
+}
+
+.code {
+  color: #616070;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card-conteiner__code {
+  margin-top: 4px;
+  margin-left: 16px;
+}
+
+.review {
+  display: flex;
+  justify-content: space-between;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card-conteiner__review {
```

Comment:
Check the 9th point in the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)